Add kicking stats to NCAAF players

AidanGlickman opened the pull request:
Fixes #373 

Adds kicking stats (attempts and percentage) to NCAAF players.

roclark commented on sportsreference/ncaaf/player.py:
```diff
@@ -501,10 +508,41 @@ def extra_points_made(self):
+    @_float_property_decorator
+    def extra_point_percentage(self):
+        """
+        Returns an ``float`` of the percentage of extra points the player made.
```
Returns **a** `float` (as opposed to `an`).

roclark commented on sportsreference/ncaaf/player.py:
```diff
@@ -501,10 +508,41 @@ def extra_points_made(self):
+    @_float_property_decorator
+    def field_goal_percentage(self):
+        """
+        Returns an ``float`` of the percentage of field goals the player made.
```
Returns **a** `float` again here.

roclark commented on sportsreference/ncaaf/player.py:
```diff
@@ -501,10 +508,41 @@ def extra_points_made(self):
+    def field_goal_percentage(self):
+        """
+        Returns an ``float`` of the percentage of field goals the player made.
+        from any distance.
```
Same as before, adding a note about the percentage range would be useful.

AidanGlickman commented on sportsreference/ncaaf/player.py:
```diff
@@ -68,6 +68,7 @@ class AbstractPlayer:
         page. If the player appears in multiple tables, all of their
         information will appear in one single string concatenated together.
     """
+
```
I think this came from `autopep8`, which I'm pretty sure is based on pycodestyle. Removed.

AidanGlickman commented:
Hi @roclark thanks for the kind words! I squashed the two commits and fixed the specific issues you mentioned. I think the testing suite needs to be changed a bit more structurally to account for these specific stats as the example html file in the integration test does not seem to have the `kicking` table. That probably warrants a new PR.

AidanGlickman commented:
git is acting up now... gonna need to figure out why it unsquashed my commits

AidanGlickman commented:
not sure what happened here, but the commits seem to be empty here now? opened again as #381 because I couldn't figure it out. Please feel free to fix however you see fit.
